package.json

```diff
@@ -16,15 +16,16 @@
   "homepage": "https://github.com/dwyl/abase#readme",
   "devDependencies": {
     "eslint": "^3.5.0",
+    "hapi": "^15.1.0",
     "istanbul": "^0.4.5",
     "pre-commit": "^1.1.3",
     "tape": "^4.6.0"
   },
   "scripts": {
-    "test": "node_modules/.bin/tape ./test/*.test.js",
+    "runtest": "node_modules/.bin/tape ./test/*.test.js",
     "lint": "node_modules/.bin/eslint .",
     "lint:fix": "node_modules/.bin/eslint . --fix",
-    "coverage": "node_modules/.bin/istanbul cover node_modules/.bin/tape ./test/*.test.js"
+    "test": "node_modules/.bin/istanbul cover node_modules/.bin/tape ./test/*.test.js"
```

Comment:
If you renamed this to test to get it to run by default on travis, a better way would be to change the script that travis runs in the `.travis.yml`. A script name of `test` for just the tests and `coverage` for the coverage makes more sense IMO.